packages/mint-components/src/components/sqm-share-code/sqm-share-code.tsx

```diff
@@ -45,11 +42,45 @@ export class ShareCode {
     attribute: "tooltip-lifespan",
   })
   tooltiplifespan: number = 1000;
+  /**
+   * Change the text alignment
+   *
+   * @uiName Share code alignment
+   * @uiType string
+   * @uiEnum ["left", "center", "right"]
+   * @uiEnumNames ["Left", "Center", "Right"]
+   */
+  @Prop({
+    attribute: "text-align",
+  })
+  textAlign: "left" | "center" | "right" = "left";
+
+  /**
+   * @uiName Copy button label
+   */
+  @Prop({
+    attribute: "copy-button-label",
+  })
+  copyButtonLabel?: string = "Copy Code";
+
+  /**
+   * Set the copy button style and placement.
```

Comment:
Looking at other components we usually don't have periods at the end of descriptions (but there is the odd exception in some existing components)